pagefaultgames/pokerogue
# [Feature] Implements Conversion 2

## What are the changes?

Implements the move Conversion 2

## Why am I doing these changes?



To be able to use Conversion 2

## What did change?


Created a function to getTypeResistances based on an input type. Implemented logic for Conversion 2 in ResistLastMoveTypeAttr based on wiki: https://bulbapedia.bulbagarden.net/wiki/Conversion_2_(move)

### Screenshots/Videos



https://github.com/pagefaultgames/pokerogue/assets/62030095/efeb7db1-5ad7-446d-bad4-3f8cb99dc145

## How to test the changes?




Give the opponent various moves with different types. Give your pokemon Conversion 2. Test with moves and make sure your pokemon's new type is a resistance to the opponent's most recent move.

## Checklist
